src/main/java/com/cbfacademy/apiassessment/DestinationAddress.java

```diff
@@ -0,0 +1,32 @@
+package com.cbfacademy.apiassessment;
+
+public class DestinationAddress {
+    private int id;
+    private String name;
+    private String address;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
```

Comment:
This is a code smell. You wouldn't want to allow the id for an address to be changed, so you should add a constructor that accepts an id and modify the `DestinationAddressService` to create a new object rather than modifying the one provided from the request.